core/services/install-services.sh

```diff
@@ -36,6 +36,9 @@ cd /home/pi/services/versionchooser/ && python3 setup.py install
 # Ping service:
 cd /home/pi/services/ping/ && python3 setup.py install
 
+# Commonwealth library:
+cd /home/pi/services/commonwealth/ && python3 setup.py install
```

Comment:
why this is running after all services if it appears as base dependence for all of them ?

Reply:
Moving up.